vpdq/python/test_vpdq_hash.py

```diff
@@ -0,0 +1,45 @@
+import vpdq
+import typing as t
+import pytest
+import test_util
+import os
+
+SAMPLE_HASH_FOLDER = "/vpdq/sample-hashes"
+SAMPLE_VIDEOS = "/tmk/sample-videos"
+FFMPEG = "/usr/bin/ffmpeg"  # please customize according to your installation (but do not commit)
+SECOND_PER_HASH = 1
+DISTANCE_TOLERANCE = 10
+QUALITY_TOLERANCE = 50
+
+
+class TestVPDQHash:
+    def test_VPDQ_hash(self):
+        d = os.path.dirname(os.path.dirname(os.getcwd()))
+        hash_folder = d + SAMPLE_HASH_FOLDER
```

Comment:
nit: instead of just + for string concat can you use? 
https://docs.python.org/3/library/pathlib.html#pathlib.Path